source/common/config/filter_json.cc

```diff
@@ -209,6 +209,7 @@ void FilterJson::translateHttpConnectionManager(
 
   JSON_UTIL_SET_BOOL(json_config, proto_config, use_remote_address);
   JSON_UTIL_SET_BOOL(json_config, proto_config, generate_request_id);
+  JSON_UTIL_SET_BOOL(json_config, proto_config, bugfix_reverse_encode_order);
```

Comment:
Why is this needed? This is just for v1 config, which is now deprecated.

Reply:
Removed.